src/Gui/PreferencePackTemplates/Behavior/Main_window_layout.cfg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<FCParameters>
+  <FCParamGroup Name="Root">
+    <FCParamGroup Name="BaseApp">
+      <FCParamGroup Name="MainWindow">
+        <FCParamGroup Name="DockWindows">
+          <FCBool Name="Std_SelectionView" Value="0"/>
+          <FCBool Name="Std_ComboView" Value="1"/>
+          <FCBool Name="Std_ReportView" Value="0"/>
+          <FCBool Name="Std_PythonView" Value="0"/>
```

Comment:
Worth listing 'Tree View', 'Property View' and 'DAG View' ?

Reply:
Yeah, probably. Right now that file is purely a skeleton for testing, I'd love some help fleshing it it, and creating other Behavior templates.

Reply:
I have added those three, but will leave this conversation open in case other additions come up.

Reply:
At the suggestion of Forum member jnxd I have added Editor font information into a template file.